test/functional/gridfs_stream.test.js

```diff
@@ -21,66 +19,64 @@ describe('GridFS Stream', function () {
    * @example-class GridFSBucket
    * @example-method openUploadStream
    */
-  it('shouldUploadFromFileStream', {
+  it('should upload from file stream', {
     metadata: { requires: { topology: ['single'] } },
 
-    test: function (done) {
-      var configuration = this.configuration;
-      var client = configuration.newClient(configuration.writeConcernMax(), { maxPoolSize: 1 });
+    test(done) {
+      const configuration = this.configuration;
+      const client = configuration.newClient(configuration.writeConcernMax(), { maxPoolSize: 1 });
 
-      // LINE var MongoClient = require('mongodb').MongoClient,
+      // LINE const MongoClient = require('mongodb').MongoClient,
```

Comment:
I really don't think these comments serve any function beyond being confusing... can we just get rid of them?